.changeset/tender-carrots-care.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-notifications-backend': minor
+'@backstage/plugin-notifications': minor
+'@backstage/plugin-notifications-common': patch
+---
+
+Notifications frontend has been updated towards expectations of the BEP 001
```

Comment:
Better explain the actual change a bit more here, this doesn't really explain the change to users. Doesn't need to be super detailed at this point tbh, just a short snipped about the removal of done state etc.